faf.spec.in

```diff
@@ -21,23 +27,39 @@ Source0: https://github.com/abrt/%{name}/archive/%{version}/%{name}-%{version}.t
 
 BuildArch: noarch
 
+%if 0%{with python3}
+%global satyr_dep python3-satyr >= 0.26
+%else
 %if 0%{?rhel} == 7
 %global satyr_dep satyr-python > 0.13-1
 %else
 %global satyr_dep satyr-python >= 0.16
 %endif
-
+%endif
 
 Requires(pre): shadow-utils
 
 Requires: postgresql
+
+%if %{with python3}
+Requires: python3
+Requires: python3-setuptools
+Requires: python3-psycopg2 >= 2.5
+Requires: python3-sqlalchemy >= 0.8
```

Comment:
python3-psycopg2 < 2.5 and python3-sqlalchemy < 0.8 does not exist. You can remove the version from the requires.